arangod/Transaction/Manager.cpp

```diff
@@ -1018,19 +1026,21 @@ Result Manager::statusChangeWithTimeout(TransactionId tid,
       // timeout
       break;
     }
-    std::this_thread::yield();
+    co_await sleepUsingSchedulerIfAvailable("trx-status-change-backup",
+                                            std::chrono::milliseconds{10});
```

Comment:
Previously we did a `this_thread::yield()` here and then continued in the same thread if the status cannot be changed.
Now we are sleeping 10ms here, and we are using the scheduler for continuations. That may take substantially longer than in the old implementation if there is a lot of activity in the scheduler.
Thus I think we should also increase the `maxWaitTime` (line 1015) from 3s to some substantially higher value here.